src/ConfirmApplyPlugin.groovy

```diff
@@ -14,18 +15,18 @@ class ConfirmApplyPlugin implements TerraformEnvironmentStagePlugin {
     @Override
     public void apply(TerraformEnvironmentStage stage) {
         if (enabled) {
-            stage.decorate(CONFIRM, addConfirmation())
+            stage.decorate(CONFIRM, addConfirmation(command))
         }
     }
 
-    public static Closure addConfirmation() {
+    public static Closure addConfirmation(String command_name) {
         return { closure ->
             // ask for human input
             try {
                 timeout(time: 15, unit: 'MINUTES') {
                     input(
-                        message: 'Are you absolutely sure the plan above is correct, and should be IMMEDIATELY DEPLOYED via "terraform apply"?',
-                        ok: 'Run terraform APPLY now',
+                        message: "Are you absolutely sure the plan above is correct, and should be IMMEDIATELY DEPLOYED via \"terraform ${command_name}\"?",
+                        ok: "Run terraform ${command_name.toUpperCase()} now",
                         submitterParameter: 'approver'
```

Comment:
Ooohh, this is super clever.  What if  we took it a  step further?

```
class ConfirmApplyPlugin...
   ...
   private  confirmMessage = 'Are you absolutely sure...' <--  the stuff that's currently hard coded.
   ...
        input (
            message: confirmMessage,
            ok: okMessage,
            submitterParameter: submitterParameter
        )
    ....
    public static withConfirmMessage(String newMessage) { this.confirmMessage = newMessage }
    public static withOkMessage(String newMessage) { this.okayMessage = newMessage }
    public static withSubmitterParameter(String newParam) { this.submitterParameter = newParam }
```

Now this has nothing to do with the DestroyPlugin.  *ANY* plugin that may need to manipulate confirmations can do so.